### PR TITLE
Added new youtube playlist

### LINKED ISSUE #869: [Add] <New TensorFlow yt channel>
### Site URL

https://youtu.be/Mubj_fqiAv8

### Description of the above provided site

Will be helpful all AI and Machine learning enthusiasts .

### Category

Youtube

### Sub Category

TensorFlow

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Abhinavcode13**
Added a new resource

## Fixes Issue
closes #869 

## Changes proposed

Added new yt playlist for learning tensorflow

### CODE REVIEW

**Comment by Abhinavcode13** [comment]
Please review the PR @CBID2 & @rupali-codes

File: database/youtube/tensorflow.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
         "category": "youtube",
         "subcategory": "tensorflow",
         "language": "english"
+    },
+        {
+        "name": "Codebasics",
+        "description": "A complete playlist for learning Tensorflow from beginning.",
```
Comment (CBID2) [inline database/youtube/tensorflow.json:20]: [Proofreading]
```suggestion
        "description": "A complete playlist for learning Tensorflow.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're so close @Abhinavcode13. I just pointed out a proofreading error in my inline comment.

**Comment by Abhinavcode13** [comment]
thanks @CBID2

File: database/youtube/tensorflow.json [hunk 2]
```diff
@@ -14,5 +14,13 @@
         "category": "youtube",
         "subcategory": "tensorflow",
         "language": "english"
+    },
+        {
```
Comment (rupali-codes) [inline database/youtube/tensorflow.json:18]: format this braces :)

**Comment by Abhinavcode13** [comment]
@rupali-codes & @CBID2 please check now

**Comment by Abhinavcode13** [comment]
@rupali-codes @CBID2 ??

**Comment by rupali-codes** [comment]
@Abhinavcode13 could you please add _closes #issuebumber_ in the description of this PR?

**Review by ujjawaltyagii — APPROVED** [review]
lgtm !

**Comment by Abhinavcode13** [comment]
Can you merge @ujjawaltyagii ?

**Pull request merged**